DOCSUP-12543: Edit and translate to Russian (bin and unbin functions) (#28194)
ClickHouse/ClickHouse: docs/en/sql-reference/functions/encoding-functions.md

## unbin {#unbinstr}

Interprets each pair of binary digits (in the argument) as a number and converts it to the byte represented by the number. The functions performs the opposite operation to [bin](#bin).

**Syntax**

``` sql
unbin(arg)
```

Alias: `UNBIN`.

For a numeric argument `unbin()` does not return the inverse of `bin()`. If you want to convert the result to a number, you can use the [reverse](../../sql-reference/functions/string-functions.md#reverse) and [reinterpretAs<Type>](../../sql-reference/functions/type-conversion-functions.md#reinterpretasuint8163264) functions.

!!! note "Note"
    If `unbin` is invoked from within the `clickhouse-client`, binary strings are displayed using UTF-8.

> Supports binary digits `0-1`. The number of binary digits does not have to be multiples of eight. If the argument string contains anything other than binary digits, some implementation-defined result is returned (an exception isn’t thrown). 

Reviewer: ```suggestion
Supports binary digits `0` and `1`. The number of binary digits does not have to be multiples of eight. If the argument string contains anything other than binary digits, some implementation-defined result is returned (an exception isn’t thrown). 
```

## unbin {#unbinstr}

> Interprets each pair of binary digits (in the argument) as a number and converts it to the byte represented by the number. The functions performs the opposite operation of [bin](#bin).

Reviewer 2: ```suggestion
Interprets each pair of binary digits (in the argument) as a number and converts it to the byte represented by the number. The functions performs the opposite operation to [bin](#bin).
```

**Syntax**

``` sql
unbin(arg)
```

Alias: `UNBIN`.

> If you want to convert the result to a number, you can use the [reverse](../../sql-reference/functions/string-functions.md#reverse) and [reinterpretAs<Type>](../../sql-reference/functions/type-conversion-functions.md#type-conversion-functions) functions.

Reviewer 2: ```suggestion
For a numeric argument `unbin()` does not return the inverse of `bin()`. If you want to convert the result to a number, you can use the [reverse](../../sql-reference/functions/string-functions.md#reverse) and [reinterpretAs<Type>](../../sql-reference/functions/type-conversion-functions.md#type-conversion-functions) functions.
```

!!! note "Note"

>     If `unbin` is invoked from within the `clickhouse-client`, binary strings display using UTF-8.

Reviewer 2: ```suggestion
    If `unbin` is invoked from within the `clickhouse-client`, binary strings are displayed using UTF-8.
```

> Supports binary digits `0-1`. The number of binary digits does not have to be multiples of eight. If the argument string contains anything other than binary digits, some implementation-defined result is returned (an exception isn’t thrown). For a numeric argument the inverse of `bin(N)` is not performed by `unbin()`.

Reviewer 2: ```suggestion
Supports binary digits `0-1`. The number of binary digits does not have to be multiples of eight. If the argument string contains anything other than binary digits, some implementation-defined result is returned (an exception isn’t thrown). 
```
